pkg/interactor/save_data_test.go

```diff
@@ -0,0 +1,113 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	thingentities "github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+var date = time.Date(2020, time.April, 15, 0, 0, 0, 0, time.UTC)
```

Comment:
Are you assuming the `timestamp` parameter will be always correct?

Reply:
Actually, the date is used only for comparison reasons. I use it as the timestamp to "save" the data on the database.

Reply:
Yeah, but should we pass this `timestamp` as an interactor parameter? Couldn't it be generated in the `save data` interactor itself?

Notice if it continues being passed through interactor we have to test it very well.

Reply:
I agree with you, I can't find any reason to generate it on the controller. Please, let me know when you change it.

Reply:
Done!